.drone.yml

```diff
@@ -0,0 +1,34 @@
+kind: pipeline
+name: default
+
+steps:
+  - name: test app
+    image: docker
+    commands:
+      - docker build -t app-test -f Dockerfile-test .
+      - docker run app-test
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+    when:
+      branch:
+        - master
+      event:
+        - pull_request
+
+  - name: report code coverage
+    image: docker
+    environment:
+      CODECOV_TOKEN:
+        from_secret: CODECOV_TOKEN
+    commands:
+      - docker build -t app-report-coverage --build-arg CODECOV_TOKEN=$CODECOV_TOKEN -f Dockerfile-coverage .
+      - docker run app-report-coverage
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+    when:
+      branch:
+        - master
+      event:
+        - pull_request
```

Comment:
We need to configure global volumes in order for Drone to access Docker: https://github.com/short-d/short/blob/bd78e93049b977753b0f3ff2ffd99e80653f9469/.drone.yml#L238